modules/bridBidAdapter.js

```diff
@@ -138,92 +128,89 @@ export const spec = {
+            meta: {
+              advertiserDomains: bid.adomain || []
+            }
+          };
 
-        if (vastUrl || vastXml) {
-          bidResponse.mediaType = VIDEO;
-          if (vastUrl) bidResponse.vastUrl = vastUrl;
-          if (vastXml) bidResponse.vastXml = vastXml;
-        } else {
-          bidResponse.ad = ad;
-          bidResponse.adUrl = adUrl;
-        };
+          if (vastUrl || vastXml) {
+            bidResponse.mediaType = VIDEO;
+            if (vastUrl) bidResponse.vastUrl = vastUrl;
+            if (vastXml) bidResponse.vastXml = vastXml;
+          } else {
+            bidResponse.ad = ad;
+            bidResponse.adUrl = adUrl;
+          };
 
-        bidResponses.push(bidResponse);
+          if (!highestBid || highestBid.cpm < bidResponse.cpm) {
+            highestBid = bidResponse;
+          }
+        });
       });
-    });
+    }
 
-    return bidResponses;
+    return highestBid ? [highestBid] : [];
   },
 
-}
+  /**
+   * Determine the user sync type (either 'iframe' or 'image') based on syncOptions.
+   * Construct the sync URL by appending required query parameters such as gdpr, ccpa, and coppa consents.
+   * Return an array containing an object with the sync type and the constructed URL.
+   */
+  getUserSyncs: (syncOptions, serverResponses, gdprConsent, uspConsent, gppConsent) => {
+    const params = {
+      endpoint: 'brid'
+    };
+
+    // Attaching GDPR Consent Params in UserSync url
+    if (gdprConsent) {
+      params.gdpr = (gdprConsent.gdprApplies ? 1 : 0);
+      params.gdpr_consent = encodeURIComponent(gdprConsent.consentString || '');
+    }
+
+    // CCPA
+    if (uspConsent && typeof uspConsent === 'string') {
+      params.us_privacy = encodeURIComponent(uspConsent);
+    }
+
+    // GPP Consent
+    if (gppConsent?.gppString && gppConsent?.applicableSections?.length) {
+      params.gpp = encodeURIComponent(gppConsent.gppString);
+      params.gpp_sid = encodeURIComponent(gppConsent?.applicableSections?.join(','));
+    }
+
+    const queryParams = Object.keys(params).length > 0 ? formatQS(params) : '';
+    let response = [];
+    if (syncOptions.iframeEnabled) {
+      response = [{
+        type: 'iframe',
+        url: SYNC_URL + 'load-cookie.html?' + queryParams
+      }];
+    }
+
+    return response;
+  }
 
-// /**
-//  * Helper function to get ad
-//  *
-//  * @param {object} bid The bid.
-//  * @return {object} ad object.
-//  */
-// function getAd(bid) {
-//   let ad, adUrl, vastXml, vastUrl;
-
-//   switch (deepAccess(bid, 'ext.prebid.type')) {
-//     case VIDEO:
-//       if (bid.adm.substr(0, 4) === 'http') {
-//         vastUrl = bid.adm;
-//       } else {
-//         vastXml = bid.adm;
-//       };
-//       break;
-//     default:
-//       if (bid.adm && bid.nurl) {
-//         ad = bid.adm;
-//         ad += createTrackPixelHtml(decodeURIComponent(bid.nurl));
-//       } else if (bid.adm) {
-//         ad = bid.adm;
-//       } else if (bid.nurl) {
```

Comment:
Do you need to keep commented code ?

Why not just delete the commented code ?